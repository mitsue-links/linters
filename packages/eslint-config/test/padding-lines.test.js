/* eslint strict: [2, "global"] */
'use strict';

const path = require('path');
const {lintFile} = require('./helper');

test('padding-lines_ok.js', async () => {
    const filePath = 'padding-lines_ok.js';
    const data = await lintFile(path.join('fixtures', filePath));

    expect(data).toStrictEqual([{
        filePath,
        messages: [],
    }]);
});

test('padding-lines_error.js', async () => {
    const filePath = 'padding-lines_error.js';
    const data = await lintFile(path.join('fixtures', filePath));

    expect(data).toMatchSnapshot();
});
