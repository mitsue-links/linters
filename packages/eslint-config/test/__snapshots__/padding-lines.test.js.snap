// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`padding-lines_error.js 1`] = `
Array [
  Object {
    "filePath": "padding-lines_error.js",
    "messages": Array [
      Object {
        "line": 15,
        "ruleId": "vars-on-top",
        "severity": 2,
      },
      Object {
        "line": 16,
        "ruleId": "padding-line-between-statements",
        "severity": 2,
      },
      Object {
        "line": 19,
        "ruleId": "vars-on-top",
        "severity": 2,
      },
      Object {
        "line": 20,
        "ruleId": "padding-line-between-statements",
        "severity": 2,
      },
    ],
  },
]
`;
